index.html

```diff
@@ -1,63 +1,148 @@
-                <li class="common-li">Bootstrap</li>
-                <li class="common-li">Ruby on Rails</li>
-                <li class="common-li">RSPec</li>
-                <li class="common-li">CapyBara</li>
-                <li class="common-li">Selenium</li>
-            </ul>
-            <ul>
-                <li class="blue1">Skills</li>
-                <li class="common-li">Codekit</li>
-                <li class="common-li">Github</li>
-                <li class="common-li">Codepen</li>
-                <li class="common-li">Gitlab</li>
-                <li class="common-li">Terminal</li>
-            </ul>
-            </div>
-            <hr class="line">
-            <ul class="icons">
-                <li><a href="#"><i class="fa fa-github bars"></i></a></li>
-                <li><a href="#"><i class="fa fa-twitter bars"></i></a></li>
-                <li><a href="#"><i class="fa fa-linkedin bars"></i></a></li>
-                <li><a href="#"><i class="fa fa-angellist bars"></i></a></li>
-                <li><a href="#"><i class="fa fa-medium bars"></i></a></li>
-            </ul>
-        </div>      
-    </body>
+
+<body>
+  <header class="nav">
+    <a class="logo" href="#">Nicholas</a>
+    <nav>
+     <i class="fas fa-bars bars"></i>
+ </nav>
+</header>
+<section class="headers">
+     <h1>Nicholas Mutale</h1> 
+ <h2>Software Developer</h2>
+ <p>I can help you build a product , feature or 
+     website Look through some of my work and experience! 
+     If you like what you see and have a project you need coded, 
+     don’t hestiate to contact me.</p>
+     <hr class="line">
+     <div class="software">
+     <ul>
+         <li class="blue1">Language</li>
+         <li class="common-li">Javascript</li>
+         <li class="common-li">Ruby</li>
+         <li class="common-li">Html</li>
+         <li class="common-li">Css</li>
+     </ul>
+     <ul>
+         <li class="blue1">Frameworks</li>
+         <li class="common-li">Bootstrap</li>
+         <li class="common-li">Ruby on Rails</li>
+         <li class="common-li">RSPec</li>
+         <li class="common-li">CapyBara</li>
+         <li class="common-li">Selenium</li>
+     </ul>
+     <ul>
+         <li class="blue1">Skills</li>
+         <li class="common-li">Codekit</li>
+         <li class="common-li">Github</li>
+         <li class="common-li">Codepen</li>
+         <li class="common-li">Gitlab</li>
+         <li class="common-li">Terminal</li>
+     </ul>
+     </div>
+     <hr class="line">
+     <ul class="icons">
+         <li><a href="#"><i class="fa fa-github bars"></i></a></li>
+         <li><a href="#"><i class="fa fa-twitter bars"></i></a></li>
+         <li><a href="#"><i class="fa fa-linkedin bars"></i></a></li>
+         <li><a href="#"><i class="fa fa-angellist bars"></i></a></li>
+         <li><a href="#"><i class="fa fa-medium bars"></i></a></li>
+     </ul>
+     <div><span>&#8595; Work</span></div>
+</section>      
```

Comment:
#### OPTIONAL

This is not part of the scope of this review so I'll mark it as a suggestion so you can implement it if you wish 🧐

There are some issues with the margins in this section as you will see in the following image 😥 Some gaps are appearing on the sides of the browser screen so you can find a way to fix it so your portfolio looks even more 🔥🔥🔥

![image](https://user-images.githubusercontent.com/15807343/148653618-a2ee7069-cd30-4e1f-b2ed-89af8d4b27aa.png)